pkg/services/preference/prefimpl/pref.go

```diff
@@ -22,7 +22,7 @@ func ProvideService(db db.DB, cfg *setting.Cfg, features *featuremgmt.FeatureMan
 		cfg:      cfg,
 		features: features,
 	}
-	if cfg.IsFeatureToggleEnabled("newDBLibrary") {
+	if cfg.IsFeatureToggleEnabled(featuremgmt.FlagNewDBLibrary) {
```

Comment:
nice refractory, I thought the feature manager is only used for frontend shared feature toggles, apparently I was wrong ;)